File: 1024hw/hw11.c
#include <stdio.h>
#include <windows.h>

int main() {
	float euro, dollar;
	float ero, dol;
	euro = 0; dollar = 0; ero = 0.0; dol = 0.0;
	printf("How much Euro do you want to exchange? ");
	scanf_s("%f",&euro);
	printf("What is the exchange rate? ");
	scanf_s("%f",&ero);
	dol = 1127.39;
	dollar = euro * ero / dol;
	printf("%.2f Euros at an exchange rate of %.2f is %.2f dollars. ",euro,ero,dollar);
	system("pause");

}

File: 1024java/hw11.java
import java.util.Scanner;

public class hw11 {
    public static void main(String args[]){
        Scanner scan = new Scanner(System.in);

        System.out.print("How many Euros are you exchanging? ");
        double euro = scan.nextDouble();
        System.out.print("What is the exchange rate ? ");
        double rate = scan.nextDouble();
        double ratedo = 1128.66817;
        double result =  euro * rate / ratedo;
        System.out.printf("%.2f euros at an exchange rate of %.2f is $%.2f",euro,rate,result);
    }
}
